streams/src/main/java/org/apache/kafka/streams/kstream/internals/WindowedDeserializer.java

```diff
@@ -33,16 +34,26 @@
 public class WindowedDeserializer<T> implements Deserializer<Windowed<T>> {
 
     private static final int TIMESTAMP_SIZE = 8;
-
+    private final Long windowSize;
+    
     private Deserializer<T> inner;
```

Comment:
add `final` (to put an additional improvement)

Reply:
A final can't be added to inner because inner would also be reassigned in the reconfig method.

Reply:
Ack.